packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -219,6 +220,12 @@ export interface BasicCrawlerOptions<Context extends CrawlingContext = BasicCraw
      */
     maxRequestRetries?: number;
 
+    /**
+     * Indicates how much time wait before crawling same domain request
+     * @default 0
+     */
+    sameDomainDelay?: number;
```

Comment:
From the interface, I cannot figure out whether this is in seconds or milliseconds... Let's maybe do `sameDomainDelayMillis` or `sameDomainDelaySecs` - so it matches how we did it with (most of) the rest of the time-based options?

Reply:
right, this should be also noted in the comment, now its milliseconds, but pretty much all of our user-facing options are in seconds, so we should probably use that instead